apps/meteor/client/views/room/RoomAnnouncement/RoomAnnouncement.tsx

```diff
@@ -8,12 +8,12 @@ import GenericModal from '../../../components/GenericModal';
 import MarkdownText from '../../../components/MarkdownText';
 import AnnouncementComponent from './AnnouncementComponent';
 
-type AnnouncementParams = {
+type RoomAnnouncementParams = {
 	announcement: string;
 	announcementDetails?: () => void;
 };
 
-const Announcement: FC<AnnouncementParams> = ({ announcement, announcementDetails }) => {
+export const RoomAnnouncement: FC<RoomAnnouncementParams> = ({ announcement, announcementDetails }) => {
```

Comment:
why not export default anymore?

Reply:
I changed in case I needed to export other components from this directory, I find it cleaner to import multiple components destructuring. should I change it back?